### PR TITLE
Update viewer screenshot in docs and make more prominent on napari.org

### PR DESCRIPTION
**Pull request by alisterburt**
# Description
I've been slightly frustrated recently with the fact that the viewer itself is so... not prominent? on napari.org. Even when you get there the screenshot is out of date, the UI has improved quite a lot! This PR updates the screenshot to an updated version of the same image and adds a copy to near the top of the index page.

Notes:
- I haven't LFS'd anything but it looks like the existing file is LFS'd... I don't know what the current state of this is and there is no guidance in the contributing guidelines about how to deal with images
- the screenshot we use on the readme currently differs from the screenshot we use on napari.org c.f. #3823. I have chosen to stay consistent with the existing site but can easily switch to the cells3d image if that's preferred (I don't prefer it personally). 

cc @melissawm @neuromusic @DragaDoncila @kne42 as docs people and @napari/core-devs as this is prominent on napari.org

before - old screenshot halfway down the homepage:
image

after - new screenshot, near the top of the homepage:
image

### CODE REVIEW

**Comment by Czaki** [comment]
There are two copies of the same image in the readme. This is your intention or did you forget to remove it? Two copies of the same image in readme do not look great to me.

**Comment by alisterburt** [comment]
Yep, I didn't want to remove the example and I didn't want to add another image so that's what I went with - we could easily go for a different image! Do you agree that the viewer should be more prominent?

File: docs/index.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ the Python ecosystem, **napari** can be easily coupled to leading machine
 learning and image analysis tools (e.g. `scikit-image`, `scikit-learn`,
 `TensorFlow`, `PyTorch`), enabling more user-friendly automated analysis.
 
+![image](./images/screenshot-add-image.png)
```
Comment (Czaki) [inline docs/index.md]:
```suggestion
![image](./images/multichannel_cells.png)
```
maybe use this?

Comment (alisterburt) [reply docs/index.md]: yep, could do this! If we go with this, I will update that screenshot too - do you know anything about the LFS situation with the docs?

Comment (Czaki) [reply docs/index.md]: The best person to ask is @jni but if I good remember now we should push every image using LFS until a separate repo for docs will be created.

**Comment by kandarpksk** [comment]
I like the idea of showing a screenshot above the "fold" so users don't have to scroll to see it. Pushing the installation section a bit down doesn't seem problematic (also as there is a list of sections on the right which hopefully maintains visibility).

It would be great to show a screenshot where a multidimensional image is being viewed in napari rather than a 2D image, maybe something like the 'Cells' sample image. I've attached a screenshot without a shadow below; I can attach one with a shadow if we prefer!

PS: I see that @Czaki already suggested a similar image!

image

**Comment by alisterburt** [comment]
Thank you for the input @kandarpksk ! Lots of valid points - I don't love the angle of the cells in that screenshot so will make another and add that to the PR

**Comment by kandarpksk** [comment]
Thanks @alisterburt! Yeah, I was merely trying to illustrate the 3D viewing capabilities (rather than show a flat view) but I have no idea what's a good angle :)

Somewhat related: does napari show the name of the file that's opened anywhere? I realized that the screenshot by itself doesn't show which (example) file is open.

**Comment by GenevieveBuckley** [comment]
Solidly agree that a nice screenshot of the viewer needs to be the first, most prominent thing you see. Thanks for doing this!

Separately, it was my impression that the larger executable docs effort should help fix these issues (by auto-generating all screenshots, so the UI of the viewer is always up to date). Is there a technical reason that's not what's happening here? cc @neuromusic since he's more up to date with the overall docs effort

**Review by jni — DISMISSED** [review]
@alisterburt can you install git lfs and re-add the new screenshot?

```
brew install git-lfs
cd /path/to/napari/repo
git lfs install
```

You might want to *first* add a commit removing the screenshot... I don't know how to manage files that *should* be lfs but aren't... git gets super confused.

**Comment by neuromusic** [comment]
@GenevieveBuckley I'll defer to @melissawm on if/how we can get an rendered screenshot on the landing page

**Comment by alisterburt** [comment]
glad we have some consensus! Barring an update from @melissawm I will move forward with what @jni suggested, thanks for the input all! Will be great to have this in for 0.4.17 😍

**Comment by alisterburt** [comment]
@kandarpksk this is not shown in the GUI - the provenance of a layer is stored in a [`Source` object](https://github.com/napari/napari/blob/3531441de2ee581f9f9da4414c03ae26a49c1441/napari/layers/_source.py) which could be used to make this happen if we decide to go that way. I'm not aware of any discussion of this but know that some plugins (e.g. AICSImageIO) name layers with this and other metadata which seems reasonable

**Comment by alisterburt** [comment]
@jni this should be ready to go now

**Comment by melissawm** [comment]
To get the screenshot autogenerated, and this page executed as a notebook, this snippet must be added to the preamble:

```
jupytext:
  text_representation:
    extension: .md
    format_name: myst
    format_version: 0.13
    jupytext_version: 1.10.3
kernelspec:
  display_name: Python 3
  language: python
  name: python3
```

See, for example, https://raw.githubusercontent.com/napari/napari/main/docs/tutorials/fundamentals/viewer.md

**Review by Czaki — APPROVED** [review]

**Pull request merged**